newrelic_rpm.gemspec

```diff
@@ -61,5 +61,5 @@ https://github.com/newrelic/newrelic-ruby-agent/
   s.add_development_dependency 'simplecov' if RUBY_VERSION >= '2.7.0'
   s.add_development_dependency 'thor' unless ENV['CI']
   s.add_development_dependency 'warning' if RUBY_VERSION >= '2.4.0'
-  s.add_development_dependency 'yard', "#{RUBY_VERSION < '2.3.0' ? '0.9.26' : '> 0.9.26'}"
+  s.add_development_dependency 'yard', '> 0.9.26'
```

Comment:
This is absolutely the correct change to make for the scope of this PR. But looking at it now I wonder if we can just remove the version constraint entirely.

Reply:
Addressed [here](https://github.com/newrelic/newrelic-ruby-agent/pull/1778/commits/f095cc4f9ce638312a084ebd8bb7822fc7f2c3f2).